connector/connect/client/jvm/src/main/scala/org/apache/spark/ml/Pipeline.scala

```diff
@@ -17,47 +17,13 @@
 
 package org.apache.spark.ml
 
-import org.apache.spark.annotation.DeveloperApi
 import org.apache.spark.internal.Logging
 import org.apache.spark.ml.param.{ParamMap, Params}
-import org.apache.spark.sql.types.StructType
 
 /**
  * A stage in a pipeline, either an [[Estimator]] or a [[Transformer]].
  */
 abstract class PipelineStage extends Params with Logging {
 
-  /**
-   * Check transform validity and derive the output schema from the input schema.
-   *
-   * We check validity for interactions between parameters during `transformSchema` and raise an
-   * exception if any parameter value is invalid. Parameter value checks which do not depend on
-   * other parameters are handled by `Param.validate()`.
-   *
-   * Typical implementation should first conduct verification on schema change and parameter
-   * validity, including complex parameter interaction checks.
-   */
-  def transformSchema(schema: StructType): StructType
```

Comment:
why removing the `transformSchema` method?
it seems in SCSC, it can be implemented with functions in `mllib-common`

Reply:
I think we should run `transformSchema` in server side, we don't need to run it in client side.

Reply:
Similarly, in pyspark side, the `Transformer/ JavaTransformer` also does not do any schema transformation.